flink-end-to-end-tests/test-scripts/test_streaming_bucketing.sh

```diff
@@ -22,7 +22,8 @@ source "$(dirname "$0")"/common.sh
 TEST_PROGRAM_JAR=${END_TO_END_DIR}/flink-bucketing-sink-test/target/BucketingSinkTestProgram.jar
 
 # enable DEBUG logging level to retrieve truncate length later
-sed -i -e 's/#log4j.logger.org.apache.flink=INFO/log4j.logger.org.apache.flink=DEBUG/g' $FLINK_DIR/conf/log4j.properties
+echo "" >> $FLINK_DIR/conf/log4j.properties
+echo "log4j.logger.org.apache.flink.streaming.connectors.fs.bucketing.BucketingSink=DEBUG, file" >> $FLINK_DIR/conf/log4j.properties
 
```

Comment:
The only thing needed is the `echo "log4j.logger.org.apache.flink.streaming.connectors.fs.bucketing.BucketingSink=DEBUG`.

The appender is already inherited by the root logger, so the `, file` is redundant, and the `echo "" >> $FLINK_DIR/conf/log4j.properties` does not add any additional information, so it can be removed.

Reply:
1. Good catch of `, file`
2. `echo "" >> $FLINK_DIR/conf/log4j.properties` is to make sure `log4j.logger.org.apache.flink.streaming.connectors.fs.bucketing.BucketingSink=DEBUG` would be always in a new line. If case of there is no `\n` at the end of log4j.properties. Although there is a `\n` now, but I don't want to depend on it. So I prefer to keep this empty line, what do you think?

Reply:
@ifndef-SleePy I do not have a strong opinion on keeping it or not. Either way it does not add or remove any important information so we can as well keep it.